### DISCUSSION TITLE
Be aware of this scammer on Twitter

### DISCUSSION THREAD

**Discussion by Dogeislife** [dogecoin/dogecoin#2935] [General]
Many people on Twitter are posing as Elon and promoting a website to have you send them doge and receive 2x back. I fell victim to this scam and lost 5000 doge, I searched up the persons address and see that they stole over 300k doge from people. Not sure if the developers can do something about this so I am posting it here. 

![E9265529-752B-4FA6-8683-F811A401EC23](https://user-images.githubusercontent.com/104403282/165199562-a29d12d2-001c-49c6-97de-31a7651a7204.jpeg)
![7CA54D06-E008-440B-82C1-01160A17A1AE](https://user-images.githubusercontent.com/104403282/165199564-6f3af73c-c82a-4750-9b1d-18a0ce6230fe.jpeg)

**Comment by patricklodder** [comment]
I'm sorry you fell victim to this. Developers cannot practically do anything about this, we're not an issuer (there is no central power).

However, if you're in the US, you might want to report it to the FTC, here: http://www.ftc.gov/complaint - other jurisdictions may have similar agencies that would appreciate your report.

**Reply by Dogeislife** [reply]
Yes I am based in the US. When filing such a complaint is this something I would need a lawyer for?

**Reply by patricklodder** [reply]
If there is any reason for you to suspect that your complaint could be held against you, or you're not sure, then I think it would be prudent to do so, yes.
